avalon/tools/cbsceneinventory/model.py

```diff
@@ -68,7 +69,21 @@ def refresh(self):
         """Refresh the model"""
 
         host = api.registered_host()
-        items = host.ls()
+        config = api.registered_config()
+
+        items = []
+        containers = host.ls()
+        for container in containers:
+            item = container.copy()
```

Comment:
There's no reason this needs an explicit copy here. Correct?

Reply:
There used to be, haven't remove the line yet

Reply:
> There used to be, haven't remove the line yet

There should be no copy, correct, but I need to keep the line :sweat_smile: